src/App.tsx

```diff
@@ -147,24 +86,33 @@ export const App: React.FC = () => {
           <nav className="filter" data-cy="Filter">
```

Comment:
You have hardcoded value here:

![image](https://github.com/mate-academy/react_todo-app-loading-todos/assets/86923089/6e6a0abc-26e4-43bb-b5d6-95867a0aa5d9)


Use actual value